apps/angular/src/core/services/anime.service.ts

```diff
@@ -10,7 +10,7 @@ import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
 
 import { UrlService } from './url.service';
 
-/** Anime service. */
+/** Anime Service. */
```

Comment:
I guess it was correct before  🌚
```suggestion
/** Anime service. */
```